source/scss/_pf-basic-variables.scss

```diff
@@ -456,3 +457,32 @@ $tag-font-weight:           $pf-font-weight-bold;
 // Use a higher than normal value to ensure completely rounded edges when
 // customizing padding or font-size on labels.
 // $tag-pill-border-radius:    10rem !default;
+
+
+// Form states and alerts
+//
+// Define colors for form feedback states and, by default, alerts.
+
+$state-success-text:             $color-pf-black-800 !default;
```

Comment:
They do work with the !default.
I found this explanation about the usage: https://robots.thoughtbot.com/sass-default
Basically 

> This means that if the variable has already been assigned to, it won’t be re-assigned, but if it doesn’t have a value yet, it will be given one.

So I assume that no variables defined by us should have the default? I see a bunch of others with default in this file...